index.html

```diff
@@ -233,6 +231,42 @@
 
 		</main>
 		
+		<aside>
```

Comment:
Up to you, but considering the layout of your page I'd suggest "section" is a more descriptive semantic tag